cms/envs/aws.py

```diff
@@ -376,6 +376,10 @@
 # Example: {'CN': 'http://api.xuetangx.com/edx/video?s3_url='}
 VIDEO_CDN_URL = ENV_TOKENS.get('VIDEO_CDN_URL', {})
 
+# Whitelist of re-writable sources, only video sources from whitelisted domains
+# will be re-written.  Should be the "netloc" of the URL, for example, www.example.com
+VIDEO_CDN_REWRITABLE_SOURCE_DOMAINS = ENV_TOKENS.get('VIDEO_CDN_REWRITABLE_SOURCES', [])
```

Comment:
Why do the variable names not match? `VIDEO_CDN_REWRITABLE_SOURCE_DOMAINS` vs `VIDEO_CDN_REWRITABLE_SOURCES`?